ios/Runner/Info.plist

```diff
@@ -6,6 +6,8 @@
 	<array>
 		<string>im.fluffychat.app</string>
```

Comment:
why not change it?

Reply:
because this is id of TaskSchedulerPermitted